Automattic/wp-calypso
# [WordPress → Jetpack app branding] Update Get Apps section

_Context:_ Following the plans to [refocus the WordPress apps on core features](https://make.wordpress.org/mobile/2022/07/27/refocusing-the-wordpress-app-on-core-features/), we plan to move features that are specific to WordPress.com to the Jetpack app. As such, we will eventually replace all mentions of the WordPress app in Calypso with Jetpack.

#### Proposed Changes

This PR update references to the mobile app found on the Get Apps screen (found via _Me_ → _Get Apps_ in Calypso). The design has also be refreshed. 

<details>
 <summary>Here's a high-level overview of the code changes ⤵️</summary>

* The `jetpack/app-branding` feature flag has been used to limit the changes to Calypso's staging site, for now. Note, in the future, this feature flag will be removed along with all Jetpack-specific class names and WordPress-app-specific code. 
* `me/sidebar/index.jsx`: The sidebar has been updated to use the Jetpack icon from the [list of available Gridicons](http://automattic.github.io/gridicons/).
* `get-apps/index.jsx`: A body class has been added to enable the setting of a white background, plus the removal of extra default padding and other styles that don't match our desired design.
* For setting the two different background colours, the same CSS pattern was used as the one found for the WooCommerce landing page (see _My Sites_ → _WooCommerce_ for an example).
* A `getJetpackBrandedPanel()` function was created, returning the newly designed components, in both `get-apps/mobile-download-card.jsx` and `get-apps/desktop-download-card.jsx`. 
</details>

#### Screenshots

<details>
 <summary>Before/After (Desktop) ⤵️</summary>

Screenshots taken on a MacBook Pro using Google Chrome:

| Before  | After |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/2998162/199476581-51273855-5c05-4a98-bc88-acf8b52ad3ed.png" width="100%"> | <img src="https://user-images.githubusercontent.com/2998162/199476642-2b53e3b5-b59d-4bb5-8cf5-ca8aaa2f9375.png" width="100%"> |

</details>

<details>
 <summary>Before/After (Mobile) ⤵️</summary>

Screenshots taken from an iPhone XR emulator using Chrome:

| Before  | After |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/2998162/199477359-bf44a6ae-d18b-4af6-9e9d-b417c62c2033.png" width="100%"> | <img src="https://user-images.githubusercontent.com/2998162/199477206-8bf18ced-1625-4716-a253-777271fd051f.png" width="100%"> |

</details>

#### Testing Instructions

* Navigate to [the auto-generated live link](https://github.com/Automattic/wp-calypso/pull/69674#issuecomment-1299360337) then head to _Me_ → _Apps_. 
* Verify that the design displays as expected across different devices, colour schemes, etc.
* If testing on a development branch, it'd be helpful to verify that the older/current design (with WordPress app references) displays as expected when the `jetpack/app-branding` feature flag is disabled.  

#### Future Iterations

The following additional changes were included in the initial designs. I plan to address these in separate PRs, as they require additional changes:

* Add second Mac button for Apple Silicon.
* Add email input to retrieve a desktop download link from a mobile device.

#### Pre-merge Checklist
